Description: Pass "nowarn" to javac ant task to disable printing of warning
 during compilation.
--- a/src/build.xml
+++ b/src/build.xml
@@ -62,6 +62,7 @@
            includes="org/**/*.java"
            excludes="org/**/jdk15/*.java"
            deprecation="on"
+           nowarn="true"
            debug="${debug}"
            includeAntRuntime="false"
            target="${target-jvm}"
@@ -80,6 +81,7 @@
            destdir="${classes}"
            includes="org/**/jdk15/*.java"
            deprecation="on"
+           nowarn="true"
            debug="${debug}"
            includeAntRuntime="false"
            target="${target-jvm}"
